test/application/inspectors/ensureNotRevokedByList.test.js

```diff
@@ -0,0 +1,22 @@
+import { ensureNotRevokedByList } from '../../../src/inspectors';
+import revokedAssertionsFixture from '../../application/domain/verifier/useCases/fixtures/revokedAssertionsFixture';
+
+describe('Inspectors test suite', function () {
+  describe('ensureNotRevokedByList method', function () {
+    describe('given it is called without revokedAssertions', function () {
+      it('should return nothing', function () {
+        expect(ensureNotRevokedByList()).toBe();
+      });
+    });
+
+    describe('given a revocation assertion matches the assertion UID', () => {
+      it('should throw an error with the revocation reason', () => {
+        const assertionUidFixture = revokedAssertionsFixture.revokedAssertions[0].id;
+        const revocationReasonAssertion = revokedAssertionsFixture.revokedAssertions[0].revocationReason;
+        expect(() => {
+          ensureNotRevokedByList(revokedAssertionsFixture.revokedAssertions, assertionUidFixture);
+        }).toThrowError(`This certificate has been revoked by the issuer. Reason given: ${revocationReasonAssertion}.`);
```

Comment:
should they throw though? A bigger concern than this PR.

Reply:
Yeah that's how the verification steps have been running since v1.. not a big fan of throwing, I want to change that eventually.